src/helpers/ProfitRepresentations.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+
+namespace hipanel\modules\stock\helpers;
+
+/**
+ * Class ProfitRepresentations
+ * @package hipanel\modules\stock\helpers
+ */
+class ProfitRepresentations
+{
+    /**
+     * @param \Closure $pack
+     * @param array $commonColumns
+     * @return array
+     */
+    public static function getColumns(\Closure $pack, array $commonColumns = []): array
```

Comment:
Maybe separate `getColumns(no args)` and `packColumns(closure, commonColumns)` ?